pkg/utils/integration/controller.go

```diff
@@ -437,3 +465,79 @@ func (h *SuiteController) GetSnapshotEnvironmentBinding(applicationName string,
 
 	return &appstudioApi.SnapshotEnvironmentBinding{}, fmt.Errorf("no SnapshotEnvironmentBinding found in environment %s %s", environment.Name, utils.GetAdditionalInfo(applicationName, namespace))
 }
+
+// FindAvailableDeploymentTargetClass attempts to find a DeploymentTargetClass with appstudioApi.Provisioner_Devsandbox as provisioner.
+func (h *SuiteController) FindAvailableDeploymentTargetClass() (*appstudioApi.DeploymentTargetClass, error) {
+	deploymentTargetClassList := &appstudioApi.DeploymentTargetClassList{}
+	err := h.KubeRest().List(context.TODO(), deploymentTargetClassList)
+	if err != nil {
+		return nil, fmt.Errorf("cannot find the avaiable DeploymentTargetClass with provisioner %s: %w", appstudioApi.Provisioner_Devsandbox, err)
```

Comment:
Shouldn't this error message be something like "error occurred while trying to list all the available DeploymentTargetClass"